samples/matter/lock/Kconfig

```diff
@@ -58,6 +58,12 @@ config APPLICATION_OTHER_LABEL
 	default "Wi-Fi" if NET_L2_OPENTHREAD
 	default "Thread" if CHIP_WIFI
 
+config THREAD_WIFI_SWITCHING_CLI_SUPPORT
+	bool "Add CLI command for triggering of the switch between Thread and Wi-Fi"
+	default n
+	depends on THREAD_WIFI_SWITCHING
+	select ENABLE_CHIP_SHELL
```

Comment:
This is ESP32 Kconfig :) We use `CHIP_LIB_SHELL`.
Btw, the dependency on Matter's shell is not really needed. You could use `SHELL_CMD_ARG_REGISTER(switch_images, NULL, "Switch between Thread and Wi-Fi", FunctionToProcessCommand, 0, 0);` directly in the application. But this is not important.